subt/docker/robotika/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+cmake_minimum_required(VERSION 3.5.1)
+project(subt_seed)
+
+if(NOT WIN32)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 -Wall -Wextra")
+endif()
+
+set (SUBT_MAJOR_VERSION 0)
+set (SUBT_MINOR_VERSION 1)
+set (SUBT_PATCH_VERSION 0)
+set (SUBT_VERSION_FULL
+  ${SUBT_MAJOR_VERSION}.${SUBT_MINOR_VERSION}.${SUBT_PATCH_VERSION})
+
+find_package(catkin REQUIRED
+  COMPONENTS
+    message_generation
+    subt_ign
+    roscpp
+)
+
+### find_package(gazebo REQUIRED)
+
+catkin_package(
+  CATKIN_DEPENDS message_runtime
+)
+
+########################
+## Message generation ##
+########################
+
+# Add here SubT ROS messages.
+
+###########
+## Build ##
+###########
+
+include_directories(
+  ${catkin_INCLUDE_DIRS}
+  ${GAZEBO_INCLUDE_DIRS}
+)
+
+link_directories(
+  ${GAZEBO_LIBRARY_DIRS}
+)
+
+## Declare a C++ executable
+add_executable(subt_seed_node src/subt_seed_node.cc)
+add_dependencies(subt_seed_node ${catkin_EXPORTED_TARGETS})
+target_link_libraries(subt_seed_node ${catkin_LIBRARIES} -lzmq)
```

Comment:
I don't understand who actually compiles the proxy.

Reply:
The proxy is copied and renamed to `subt_seed_node` and then compiled by `catkin_make install` during docker creation